Validator/Constraints/Unique.php

```diff
@@ -16,6 +16,35 @@
 #[Attribute(Attribute::TARGET_CLASS | Attribute::IS_REPEATABLE)]
 class Unique extends UniqueEntity
 {
-    /** @var string */
-    public $service = 'doctrine_odm.mongodb.unique';
+    /**
+     * @param array|string      $fields     The combination of fields that must contain unique values or a set of options
+     * @param bool|array|string $ignoreNull The combination of fields that ignore null values
+     */
+    public function __construct(
+        $fields,
+        string $message = null,
+        string $service = null,
+        string $em = null,
+        string $entityClass = null,
+        string $repositoryMethod = null,
+        string $errorPath = null,
+        bool|string|array $ignoreNull = null,
+        array $groups = null,
+        $payload = null,
+        array $options = []
+    ) {
+        parent::__construct(
+            $fields,
+            $message,
+            'doctrine_odm.mongodb.unique',
```

Comment:
```suggestion
        string $service = 'doctrine_odm.mongodb.unique',
        string $em = null,
        string $entityClass = null,
        string $repositoryMethod = null,
        string $errorPath = null,
        bool|string|array $ignoreNull = null,
        array $groups = null,
        $payload = null,
        array $options = []
    ) {
        parent::__construct(
            $fields,
            $message,
            $service,
```

This way the user can specify the service, otherwise the one from the constructor is not used